pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -166,7 +169,73 @@ let
 
   # Watch out for python <> boost compatibility
   python = python310.override {
-    packageOverrides = self: super: {
+    packageOverrides = self: super: let cryptographyOverrideVersion = "40.0.1"; in {
+      # Ceph does not support `cryptography` > 40 yet:
+      # * https://github.com/NixOS/nixpkgs/pull/281858#issuecomment-1899358602
+      # * Upstream issue: https://tracker.ceph.com/issues/63529
+      #   > Python Sub-Interpreter Model Used by ceph-mgr Incompatible With Python Modules Based on PyO3
+      #
+      # We pin the older `cryptography_40` here;
+      # this also forces us to pin an older `pyopenssl` because the current one
+      # is not compatible with older `cryptography`, see:
+      #     https://github.com/pyca/pyopenssl/blob/d9752e44127ba36041b045417af8a0bf16ec4f1e/CHANGELOG.rst#2320-2023-05-30
+      cryptography-vectors = (self.callPackage ../../../development/python-modules/cryptography/vectors.nix { }).overridePythonAttrs (old: {
+        src = fetchPypi {
+          pname = "cryptography_vectors";
+          version = cryptographyOverrideVersion;
+          hash = "sha256-hGBwa1tdDOSoVXHKM4nPiPcAu2oMYTPcn+D1ovW9oEE=";
+        };
+      });
```

Comment:
Please just disable tests in cryptography instead.

Reply:
@SuperSandro2000 Is there a drawback of running the tests? Given that they are not slow, it seems nice to check that the version/src override doesn't add any bugs.

Reply:
I've disabled tests as requested in any case with the latest force-push of this PR.